app/validators/collection_druid_validator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Validates that collection exists.
+class CollectionDruidValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return unless Settings.validate_collection_druid
+    return if record.errors[attribute].present?
+
+    cocina_obj = dor_services_client.object(value).find
+    record.errors.add attribute, 'not a collection' unless cocina_obj.type == Cocina::Models::ObjectType.collection
```

Comment:
Can we use `... unless cocina_obj.collection?` (from the `Cocina::Models::Checkable` module) or does that not cover cases we think we might see here?

Reply:
That helped me see where the existence of the collection was being checked -- thank you Mike.

Reply:
Thanks for the suggestion. Changing.